api/v1alpha3/tags.go

```diff
@@ -18,7 +18,10 @@ package v1alpha3
 
 import (
 	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
```

Comment:
Can this be refactored to avoid including the aws sdk in the types that we expose? I know it's not being exposed directly through the types we expose, but I do worry that it might make it easier to accidentally do that in the future and not realize it as easily.

Reply:
This is fixed in the recent changes addressed for @richardcase's comments. PTAL.